src/main.py

```diff
@@ -16,36 +16,22 @@
 myAccount = AccountManager(CONSUMERKEY, CONSUMERSECRET, ACCESSKEY, ACCESSSECRET)
 myAccount.login()
 
-#def main():
-# Define Command Line here in main
-
-# Function To display Owner's Details
-def DisplayOwnerAccountInfo ():
-    print ("Your Account Information :")
-    data = myAccount.getMyUserInfo()
-    for key, value in data.items():
-        print (key,value)
-
-DisplayOwnerAccountInfo()
```

Comment:
We want to reuse DisplayOwnerAccountInfo() , DisplayMyFollowers(), DisplayFollowingMe() later on for Command Line Interface . I would say better to keep them unless @smmtaheri has better idea